tests/Unit/Application/Commands/ArchiveShoppingList/ArchiveShoppingListModelTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\Unit\Application\Commands\ArchiveShoppingList;
+
+use Lindyhopchris\ShoppingList\Application\Commands\ArchiveShoppingList\ArchiveShoppingListModel;
+use PHPUnit\Framework\TestCase;
+
+class ArchiveShoppingListModelTest extends TestCase
+{
+    /**
+     * @return string
+     */
+    public function listProvider(): string
+    {
+        return 'groceries';
+    }
+    /**
+     * @param string $list
+     * @return void
+     * @dataProvider listProvider
+     */
+    public function test(string $list): void
```

Comment:
Required: you don't need to use a data provider here. Data providers are for when you want the test to run multiple times, each time with a different input. Here you only have a single input, so you do not need to use a data provider.

You should remove `$list` as it is not needed in this test. You can just use the string `groceries`.